ui/src/ducks/app/nodes.js

```diff
@@ -252,6 +252,17 @@ export function* fetchNodes() {
             node.status.conditions.find(
               (conditon) => conditon.type === 'Ready',
             );
+
+          // Store the name of conditions which the status are True in the array
+          // given the avaiable conditions (Ready, DiskPressure, MemoryPressure, PIDPressure, Network Unavailable, Unschedulable)
+          let conditions = [];
+          const activeCondition = node.status.conditions.find(
+            (cond) => cond.status === 'True',
+          );
+          if (activeCondition && activeCondition.types) {
+            conditions.push(activeCondition.type);
+          }
```

Comment:
That's only giving one condition, if any, in the `conditions` array... I guess it should be:

```suggestion
          const conditions = node.status.conditions.filter(
            (cond) => cond.status === 'True'
          ).map((cond) => cond.type);
```

Or, using a single `reduce` call (which allows to also check if `type` is a valid attribute):

```javascript
const conditions = node.status.conditions.reduce(
  (acc, cond) => {
    if (cond.status === 'True' && cond?.type)
      acc.push(cond.type);
    return acc;
  },
  []
);
```